ParserApp/models/lamoda_models.py

```diff
@@ -0,0 +1,15 @@
+from datetime import datetime
+from decimal import Decimal
+
+from pydantic import BaseModel
+
+
+class LamodaProduct(BaseModel):
+    product_sku: str
```

Comment:
What that means?

Reply:
sku is a unique identifier for every product from site